src/README.md

```diff
@@ -4,76 +4,103 @@ The purpose of this document is to describe the internal code structure and majo
 
-<a id="3"></a>
-### Client
-- DAOS Client: Resides on client node. Application that links to libdaos and talks to DAOS Agent. DAOS provides a collection of client libraries (one per-layer) that implement the external API exported to application developers. The DAOS client libraries as well as the networking parts do not spawn any internal threads and can be linked directly with the application. It supports the event and event queue interface for non-blocking operations. During execution, a client library can either call into the client library of the lower layer, or use the network transport to send a RPC to its server counterpart, as show in Figure 4 2.
+A DAOS system is identified by a system name and consists of a set of DAOS servers connected to the same fabric. Two different systems comprise two disjoint sets of servers and do not coordinate with each other. DAOS pools cannot span across multiple systems.
 
-<a href="client/README.md">DAOS Client and I/O Middleware</a>
+Internally, a DAOS server is composed of multiple daemon processes. The first one to be started is the <a href="control/README.md">control plane</a> (binary named daos_server for convenience) which is responsible for parsing the configuration file, provisionning storage and eventually starting and monitoring one or mutiple instances of the <a href="iosrv/README.md">data plane</a> (i.e. daos_io_server binary). The control plane is written in Go and implements the DAOS management API over the gRPC framework that provides a secured out-of-band channel to administrate a DAOS system. The number of data plane instances to be started by each server as well as the storage, CPU and fabric interface affinity can be configured through the YAML configuration file.
 
-<a id="4"></a>
-### Agent
-- DAOS Agent: daos_agent is a daemon process residing on the client node. Trusted intermediary in authentication of DAOS Client application. Signs DAOS Client credentials using local certificates
+The data plane is a multi-threaded process written in C that runs the DAOS storage engine. It processes incoming metadata and I/O requests though the CART communication middleware and accesses local NVM storage via the PMDK (for storage-class memory, aka SCM) and SPDK (for NVMe SSDs) libraries. The data plane relies on Argobots for event-based parallel processing and exports multiple targets that can be independently addressed via the fabric. Each data plane instance is assigned a unique rank inside a DAOS system.
+
+The control plane and data plane processes communicate locally through Unix Domain Socket and a custom lightweight protocol called dRPC.
```

Comment:
sp - "through [a] Unix Domain Socket"

Reply:
I would just pluralize Socket to sockets since there is one socket for daos_server and one for each iosrv.